package.json

```diff
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/johnpapa/lite-server#readme",
   "dependencies": {
-    "browser-sync": "^2.12.3",
+    "browser-sync": "^2.14.0",
```

Comment:
Hi @dennify 
The caret should already handle this, if you run `npm update` after the fact (eg. after an initial `npm install`.
http://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/